lib/src/Uncertainty/Process/CompositeProcess.cxx

```diff
@@ -129,7 +129,16 @@ TimeSeries CompositeProcess::getFuture(const UnsignedInteger stepNumber) const
   /* TimeGrid associated with the possible future */
   const Scalar timeStep = timeGrid.getStep();
   const RegularGrid futurTimeGrid(timeGrid.getEnd(), timeStep, stepNumber);
-  return TimeSeries(futurTimeGrid, function_(antecedent_.getFuture(stepNumber).getValues()));
+  FieldFunction function(function_);
+  try
+  {
+    function.setInputMesh(futurTimeGrid);
```

Comment:
Even if this is useless here, I suggest to also call setOutputMesh.

Reply:
Did you remove this comment ?

Reply:
No

Reply:
ok, it's just that it appeared grayed out

Reply:
ok, done